tgui/packages/tgui/interfaces/SpecMenu.js

```diff
@@ -0,0 +1,99 @@
+import { useBackend } from "../backend";
+import { Box, Button, Flex, Section, Divider } from "../components";
```

Comment:
Doesn't look like you're using `<Box>` anywhere, so safe to remove.
![image](https://user-images.githubusercontent.com/57483089/132144165-ee132bbb-587b-48e5-8965-45a6882c4626.png)
```suggestion
import { Button, Flex, Section, Divider } from "../components";
```